src/worker/ingestion/process-event.ts

```diff
@@ -38,22 +41,40 @@ export class EventsProcessor {
     posthog: ReturnType<typeof nodePostHog>
     teamManager: TeamManager
     personManager: PersonManager
+    actionManager: ActionManager
+    pubSub: PubSub
 
     constructor(pluginsServer: PluginsServer) {
+        this.ready = false
         this.pluginsServer = pluginsServer
         this.db = pluginsServer.db
         this.clickhouse = pluginsServer.clickhouse
         this.kafkaProducer = pluginsServer.kafkaProducer
         this.celery = new Client(pluginsServer.db, pluginsServer.CELERY_DEFAULT_QUEUE)
         this.teamManager = new TeamManager(pluginsServer.db)
         this.personManager = new PersonManager(pluginsServer)
+        this.actionManager = new ActionManager(pluginsServer.db)
+        this.pubSub = new PubSub(pluginsServer, {
+            'fetch-action': async (message) => await this.actionManager.fetchAction(parseInt(message)),
+            'delete-action': (message) => this.actionManager.deleteAction(parseInt(message)),
+        })
```

Comment:
The only problem with this is that it'll add another redis connection per worker thread, which can add up. I guess even if we have officially given up on Heroku free redis tiers, it might be good to reload these inside the workers with a similar `broadcastTask` system that plugin reloads use.

Reply:
Agreed, used similar system as `reloadPlugins`, avoiding increase of Redis connections with per-thread pubsub instances